qiime/sdk/workflow.py

```diff
@@ -155,6 +174,16 @@ def from_function(cls, function, inputs, outputs, name, doc):
             doc=doc, import_path=import_path, function_name=function_name,
             parameters=parameters, results=results)
         signature = Signature(name, inputs, outputs)
+
+        # just to show that we've got the views
+        input_views = {input: function.__annotations__[input]
+                       for input in inputs.keys()}
+        output_view_types = qiime.core.type.util.tuplize(
```

Comment:
that's going a bit deep. would you be supportive of the creation of an issue oriented around import aliases?